### ISSUE TITLE
Review component accessibility against WCAG 2.2 draft

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#74]
https://www.w3.org/blog/2020/08/wcag22-wide-review/

Web Content Accessibility Guidelines (WCAG) 2.2 draft will become the standard 
that we will be audited against. The Working Group hopes to publish the final version 
of WCAG 2.2 as a “W3C Recommendation” web standard in mid 2021.

We should be proactive about improving the accessibility of our components by
implementing any recomended changes ahead of publication. As its a draft standard, 
there is also an [opportunity to provide feedback](https://www.w3.org/blog/2020/08/wcag22-wide-review/).

## Review Table

The below tables notes which components have been reviewed and
lists issues of interest. _If we'd like to track process of implementing
the issues we may want a new label for the Origami project board._

Components with Sass which maintained by the core Origami team:

| component | status | issues | notes |
| --- | --- | --- | --- |
| o-assets | | | |
| o-banner | | | |
| o-big-number | | | |
| o-brand | | | |
| o-buttons | partially reviewed, fail | focus state: https://github.com/Financial-Times/o-buttons/issues/264 | continue review: add guidence around Pointer Target Spacing (how much space between buttons),  pagination buttons likely can't pass Pointer Target Spacing without changes |
| o-colors | | | |
| o-cookie-message | | | |
| o-editorial-layout | | | |
| o-editorial-typography | | | |
| o-expander | | | |
| o-fonts | | | |
| o-footer | | | |
| o-footer-services | | | |
| o-forms | | | |
| o-ft-affiliate-ribbon | | | |
| o-gallery | | | |
| o-grid | | | |
| o-header | | | |
| o-header-services | | | |
| o-icons | | | |
| o-labels | | | |
| o-layout | | | |
| o-lazy-load | | | |
| o-loading | | | |
| o-message | | | |
| o-meter | | | |
| o-normalise | | | |
| o-overlay | | | |
| o-quote | | | |
| o-share | partial review, failed | spacing: https://github.com/Financial-Times/o-share/issues/189 | continue review |
| o-spacing | | | |
| o-stepped-progress | | | |
| o-syntax-highlight | | | |
| o-table | | | |
| o-tabs | | | |
| o-teaser | | | |
| o-teaser-collection | | | |
| o-toggle | | | |
| o-tooltip | | | |
| o-typography | | | |
| o-visual-effects | | | |

Components with Sass which are not maintained by the core Origami team:

| component | status | issues | notes |
| --- | --- | --- | --- |
| o-ads | | | |
| o-comments | | | |
| o-subs-card | | | |
| o-video | | | |
